cmd/translatesfx/README.md

```diff
@@ -0,0 +1,102 @@
+# SignalFx Smart Agent Configuration Translation Tool (Experimental)
+
+This package provides a command-line tool to translate a SignalFx Smart Agent
+configuration file into an OpenTelemetry Collector configuration.
+
+## Usage
+
+The `translatesfx` command requires one argument, the signalfx configuration
```

Comment:
Not requesting now, but I think this could be a feature for a more generalized config verifier and linter.

`otelcolconfig --from-smart-agent`
`otelcolconfig --verify my_collector_config.yaml`
`otelcolconfig --build --receivers jaeger --exporters otlp` etc.